fanuc_lrmate200ic5h_moveit_config/launch/move_group.launch

```diff
@@ -5,15 +5,16 @@
   <!-- GDB Debug Option -->
   <arg name="debug" default="false" />
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
-  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
+  <arg     if="$(arg debug)" name="launch_prefix"
+	   value="gdb -x $(find fanuc_lrmate200ic5h_moveit_config)/launch/gdb_settings.gdb --ex run --args" />
```

Comment:
It's not listed in the list of files that are processed / generated by the setup assistant (see [here](https://github.com/ros-planning/moveit_setup_assistant/blob/c5eaffbacb8d9e70d775e22664e7ba96efb1cc78/src/widgets/configuration_files_widget.cpp#L196-L432)), and thus does not seem to be added to a (normal) MoveIt config pkg.

I'm not sure whether that is an oversight, or whether this is really only developer infrastructure / convenience (and thus not meant to be part of a moveit pkg).